guides/gatherers.md

```diff
@@ -203,6 +204,33 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/packageversion_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion_test.go)
 
+### saphostctrl
+
+This gatherer returns allows access to certain webmethods that `saphostctrl` implements. At this moment only `Ping` and `ListInstances` are supported.
+
+Sample arguments
```

Comment:
Instead of using this `Sample arguments` table, I think that putting the examples as `sbd_dump` does is better.
What do you think?